ElectricityBillApi.Tests/ElectricProviderTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿using ElectricityBillApi.Models;
+using System;
+using Xunit;
+
+namespace ElectricityBillApi.Tests
+{
+    public class ElectricProviderTests
+    {
+        private ElectricProvider _sut;
+
+        public ElectricProviderTests()
+        {
+            _sut = new ElectricProvider();
+        }
+
+        [Fact]
+        public void CalculatePrice_CalculatesCorrectPrice()
+        {
+            // arrange
+            var loc1 = new Location()
+            {
+                X = 0,
+                Y = 0,
+                Z = 0
+            };
+            var loc2 = new Location()
+            {
+                X = 1,
+                Y = 1,
+                Z = 1
+            };
+            decimal plantPrice = 5;
+            var plant = new PowerPlant()
+            {
+                Location = loc1,
+                ElectricityPrice = plantPrice
+            };
+
+            // act
+            _sut.Subscribe(plant);
+            var actual = _sut.CalculatePrice(new Address() { Location = loc2 });
+            var expected = 3 * plantPrice;
```

Comment:
Why `*3`?

Reply:
Added a method to calc the expected price. Makes things more clear I hope.